### PR TITLE
Customized ResNet || ImageNet and CIFAR10 Training || Lambda GPU Cloud

### PR DESCRIPTION
**Pull request by chuanli11**
Two entries from Lambda GPU cloud ($1.50/h):

ImageNet training of fastai's imagenet18 with customized learning rate scheduling.
Time to 93% Top 5 accuracy: 12.663 h
Total cost: $18.995

CIFAR10 training of distributed Baidu9Net with average pooling layer removed.
Time to 94% Top 1 accuracy: 0.0282 h
Total cost: $0.042

### CODE REVIEW

File: CIFAR10/train/lambda_BaiduNet9_lambda_cloud_single_pytorch.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+    "version": "v1.0",
+    "author": "Chuan Li",
+    "authorEmail": "c@lambdal.com",
+    "framework": "fastai / Pytorch 1.0.0",
+    "codeURL": "https://github.com/lambdal/CIFAR10-Distributed-Training-BaiduNet9P/blob/baea41d37bebb8d6504e7779bb5dff4605953707/train_cifar10_BaiduNet9P.py",
+    "model": "BaiduNet9",
+    "hardware": "Lambda GPU Cloud",
```
Comment (codyaustun) [inline CIFAR10/train/lambda_BaiduNet9_lambda_cloud_single_pytorch.json]: What is the machine type?

File: ImageNet/train/lambda_resnet50_lambda_cloud_single_pytorch.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{
+    "version": "v1.0",
+    "author": "Chuan Li",
+    "authorEmail": "c@lambdal.com",
+    "framework": "ncluster / Pytorch 1.0.0",
+    "codeURL": "https://github.com/lambdal/imagenet18/blob/1b43a01af2b8ee5bb9e2405a626d995a750edbb5/training/train_imagenet_nv.py",
+    "model": "Resnet 50",
+    "hardware": "Lambda GPU Cloud",
```
Comment (codyaustun) [inline ImageNet/train/lambda_resnet50_lambda_cloud_single_pytorch.json]: What is the machine type?

**Review by codyaustun — CHANGES_REQUESTED** [review]
Looks good overall, but please add the machine type or number/type of GPUs to the hardware field.

File: CIFAR10/train/lambda_BaiduNet9_lambda_cloud_single_pytorch.json (hunk 1 above)
Comment (chuanli11) [reply CIFAR10/train/lambda_BaiduNet9_lambda_cloud_single_pytorch.json]:
```suggestion
    "hardware": "Lambda GPU Cloud - gpu.4x instance",
```

File: ImageNet/train/lambda_resnet50_lambda_cloud_single_pytorch.json (hunk 2 above)
Comment (chuanli11) [reply ImageNet/train/lambda_resnet50_lambda_cloud_single_pytorch.json]:
```suggestion
    "hardware": "Lambda GPU Cloud - gpu.4x instance",
```

File: ImageNet/train/lambda_resnet50_lambda_cloud_single_pytorch.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{
+    "version": "v1.0",
+    "author": "Chuan Li",
+    "authorEmail": "c@lambdal.com",
+    "framework": "ncluster / Pytorch 1.0.0",
+    "codeURL": "https://github.com/lambdal/imagenet18/blob/1b43a01af2b8ee5bb9e2405a626d995a750edbb5/training/train_imagenet_nv.py",
+    "model": "Resnet 50",
+    "hardware": "Lambda GPU Cloud - gpu.4x instance",
```
Comment (codyaustun) [inline ImageNet/train/lambda_resnet50_lambda_cloud_single_pytorch.json]: Please add the type of GPU as shown below.

```suggestion
    "hardware": "Lambda GPU Cloud - 4x GTX 1080 Ti",
```

File: CIFAR10/train/lambda_BaiduNet9_lambda_cloud_single_pytorch.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
+{
+    "version": "v1.0",
+    "author": "Chuan Li",
+    "authorEmail": "c@lambdal.com",
+    "framework": "fastai / Pytorch 1.0.0",
+    "codeURL": "https://github.com/lambdal/CIFAR10-Distributed-Training-BaiduNet9P/blob/baea41d37bebb8d6504e7779bb5dff4605953707/train_cifar10_BaiduNet9P.py",
+    "model": "BaiduNet9",
+    "hardware": "Lambda GPU Cloud - gpu.4x instance",
```
Comment (codyaustun) [inline CIFAR10/train/lambda_BaiduNet9_lambda_cloud_single_pytorch.json]: Please add the type of GPU as shown below.

```suggestion
    "hardware": "Lambda GPU Cloud - 4x GTX 1080 Ti",
```

**Review by codyaustun — CHANGES_REQUESTED** [review]

File: CIFAR10/train/lambda_BaiduNet9_lambda_cloud_single_pytorch.json (hunk 4 above)
Comment (stephenbalaban) [reply CIFAR10/train/lambda_BaiduNet9_lambda_cloud_single_pytorch.json]: We've accepted these changes and updated our fork.

File: ImageNet/train/lambda_resnet50_lambda_cloud_single_pytorch.json (hunk 3 above)
Comment (stephenbalaban) [reply ImageNet/train/lambda_resnet50_lambda_cloud_single_pytorch.json]: We've accepted these changes and updated our fork.

**Comment by stephenbalaban** [comment]
@codyaustun, @chuanli11 and I've accepted your change suggestions and updated our fork.

**Review by codyaustun — APPROVED** [review]

**Comment by codyaustun** [comment]
Thanks! Looks good to me.

**Pull request merged**